app.py

```diff
@@ -30,6 +30,22 @@
     return json(ranking_matrices)
 
 
+@app.route("/ranking/<eid>", methods=["GET"])
+@requires_db
+async def list_rankings(db, request, eid):
```

Comment:
Let's change the name of this function, `list_rankings` sounds like you were going to list several ranking matrices rather than retrieving a single one.